### PR TITLE
build improvements

### PR DESCRIPTION
**Pull request by pohly**
One cosmetic change and one fix.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -86,13 +86,14 @@ ARG BIN_SUFFIX
 # which flavor (production or testing). The actual binary name in the
 # image is going to be the same, to avoid unnecessary deployment
 # differences.
-RUN make VERSION=${VERSION} pmem-csi-driver${BIN_SUFFIX} pmem-vgm${BIN_SUFFIX} pmem-ns-init${BIN_SUFFIX} && \
+RUN set -x && \
```
Comment (avalluri) [inline Dockerfile]: @pohly Is `set -x` is intentional here? I guess by default all RUN statements are echoed by `docker build`.

Comment (avalluri) [reply Dockerfile]: @pohly Ignore my earlier comment. Here I see adding `set -x` seems useful for debugging individual commands inside RUN

**Comment by avalluri** [comment]
Looks good to me.

File: Dockerfile (hunk 1 above)
Comment (pohly) [reply Dockerfile]: Exactly. In the output without `-x` it was impossible to see which command actually caused Go to fetch dependencies.

**Pull request merged**